spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/ExtensionsTestBase.java

```diff
@@ -57,6 +58,8 @@ public static void startMetastoreAndSpark() {
             .enableHiveSupport()
             .getOrCreate();
 
+    TestBase.sparkContext = JavaSparkContext.fromSparkContext(spark.sparkContext());
```

Comment:
`TestBase.spark` is created in the line above, so we should probably configure the spark context appropriately here